recipe/meta.yaml

```diff
@@ -12,31 +12,29 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: True  # [py3k and not win]
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
-    - numpy 1.11.*
+    - numpy x.x
     - setuptools
-    - toolchain
```

Comment:
This is probably not needed here. The tests will confirm that. So let's remove more variables that can interfere with the build.